stubs/6/EloquentBuilder.stubphp

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
@mr-feek Should I also add stubs for Laravel 7, or can we drop support for Laravel 7 explicitly instead?

Reply:
I'm fine with dropping support explicitly for laravel 7 since it is no longer supported by laravel. No need to bog ourself down supporting unsupported laravel versions.

Reply:
@mr-feek If the effort is reasonable, could it make sense to release one last patch release with Laravel 7 support, and then a minor release without?

Reply:
See #185.